container_registry/container_analysis/create_note.go

```diff
@@ -20,20 +20,19 @@ import (
 	"context"
 	"fmt"
 
-	containeranalysis "cloud.google.com/go/containeranalysis/apiv1beta1"
-	grafeaspb "google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/grafeas"
-	"google.golang.org/genproto/googleapis/devtools/containeranalysis/v1beta1/vulnerability"
+	containeranalysis "cloud.google.com/go/containeranalysis/apiv1"
+	grafeaspb "google.golang.org/genproto/googleapis/grafeas/v1"
 )
 
 // createNote creates and returns a new vulnerability Note.
 func createNote(noteID, projectID string) (*grafeaspb.Note, error) {
 	ctx := context.Background()
-	client, err := containeranalysis.NewGrafeasV1Beta1Client(ctx)
+	client, err := containeranalysis.NewClient(ctx)
 	if err != nil {
 		return nil, fmt.Errorf("NewGrafeasV1Beta1Client: %v", err)
 	}
 	defer client.Close()
-
+	
```

Comment:
Looks like you need to run `gofmt -s` (there's usually a way to have your editor run it for you every time you save). Kokoro failed the `gofmt` check.

If you're running it from command line, you'll want to run `gofmt -w -s .` (write files directly, do extra simplifications, all files in this directory (recursive)).

Reply:
sorry, should have caught that! fixed